src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -163,13 +167,7 @@ void init(int opmode, Key key, GCMParameterSpec spec)
         reInit = false;
 
         // always encrypt mode for embedded cipher
-        try {
-            blockCipher.init(false, key.getAlgorithm(), keyValue);
-        } finally {
-            if (!encryption) {
-                Arrays.fill(keyValue, (byte) 0);
-            }
-        }
+        blockCipher.init(false, key.getAlgorithm(), keyValue);
```

Comment:
Is this change intentional? Looks like we are reverting to older version of source and undo newer changes.

Reply:
Nope.. unintentional